omero/developers/Python.rst

```diff
@@ -39,6 +39,9 @@ In addition to the auto-generated Python libraries of the core |OmeroApi|,
 facilitates several aspects of working with the Python API, such as
 connection handling, object graph traversal and lazy loading.
 
+Building on the 'BlitzGateway', The Jackson Laboratory have created
```

Comment:
```suggestion
Building on the 'BlitzGateway', the Jackson Laboratory has created
```

Reply:
Some examples in omero-guide-python should be considered